src/helm-frontend/src/components/LeaderboardTables.tsx

```diff
@@ -37,6 +40,107 @@ export default function LeaderboardTables({
   const [filteredModels, setFilteredModels] =
     useState<string[]>(modelsToFilter);
 
+  const [schema, setSchema] = useState<Schema | undefined>(undefined);
```

Comment:
For cleanliness I can move this out into a separate component when I refactor the leaderboard next quarte.r